src/CTA.WebForms2Blazor/ControlConverters/HyperLinkControlConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using HtmlAgilityPack;
+
+namespace CTA.WebForms2Blazor.ControlConverters
+{
+    public class HyperLinkControlConverter : ControlConverter
+    {
+        private Dictionary<String, String> _hyperLinkControlsMap = new Dictionary<string, string>() 
+            {["navigateurl"] = "href", ["cssclass"] = "class"};
```

Comment:
Something like css class seems like it would hold for all controls. If so then we might want to add this to some sort of secondary general control property map or have Convert2Blazor call the base class method where general changes like that are done

Reply:
Yeah I agree, but I think for this PR I'm focusing on the overall framework and in upcoming PRs flush out the actual migration rules